### ISSUE TITLE
View Image button not appearing

### ISSUE THREAD

**Issue by riahc3** [bijij/ViewImage#251]
The View Image button not appearing. Seems Google changed something.

**Comment by FranBunnyFFXII** [comment]
Same issue here, its been like this for weeks. The button is not appearing. Getting very frustrated as I use this constantly.

**Comment by bijij** [comment]
Duplicate of #242 this has been fixed however I'm waiting on google to approve the update for chrome. Could be a few weeks unfortunately.
